testing/basic-tenant.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# This script requires: kubectl, kind, jq
+
+#set -e
+
+yell() { echo "$0: $*" >&2; }
+die() {
+  yell "$*"
+  (kind delete cluster || true ) && exit 111
+}
+try() { "$@" || die "cannot $*"; }
+
+try kind create cluster --config kind-config.yaml
+
+echo "Installing Current Operator"
+
+# TODO: Compile the current branch and create an overlay to use that image version
+try kubectl apply -k ../resources
+
+echo "Waiting for Operator Pods to come online"
+
+try kubectl wait --namespace minio-operator \
+  --for=condition=ready pod \
+  --selector=name=minio-operator \
+  --timeout=90s
+
+echo "Installing lite tenant"
+
+try kubectl apply -k ../examples/kustomization/tenant-lite
+
+
+echo "Waiting for the tenant statefulset, this indicates the tenant is being fulfilled"
+waitdone=0
+totalwait=0
+while :; do
```

Comment:
```suggestion
while true; do
```

Let's use readable values.